index.src.html

```diff
@@ -1218,16 +1255,19 @@
     5. [=Queue a control message=] to encode |frameClone|.
 
     [=Running a control message=] to encode the frame means performing these steps.
-    1. Attempt to use {{VideoEncoder/[[codec implementation]]}} to encode
+    1. If {{VideoEncoder/[[active color space]]}} is `null`, assign
+        |frame|.{{VideoFrame/colorSpace}} to
```

Comment:
Adding back Chris' concern here: `VideoFrame.colorSpace` could be `null`, although this should only happens in cases where the UA produced the frame itself and wasn't able to map to a WebCodecs color space. Potentially we should remove the nullability here, since UAs could produce a `VideoColorSpace` with null members instead.